install.sh

```diff
@@ -7,6 +7,13 @@ elif grep '^ID=.*opensuse' /etc/os-release > /dev/null; then
   sudo zypper install $(cat ./dependencies/os-linux.txt)
 elif grep '^ID=arch' /etc/os-release > /dev/null; then
   sudo pacman -S --needed $(cat ./dependencies/os-linux.txt)
+elif grep '^ID=.*fedora' /etc/os-release > /dev/null; then
+  # since Fedora 22, DNF is the default package manager
+  if command -v dnf >/dev/null 2>&1; then
+    sudo dnf install $(cat ./dependencies/os-fedora.txt)
+  else
+    sudo yum install $(cat ./dependencies/os-fedora.txt)
```

Comment:
You can do `dnf install /usr/bin/convert` so if that's also possible on other Linuxes, we can change the package list to command list.